jerry-core/ecma/builtin-objects/ecma-builtin-string-prototype.c

```diff
@@ -920,6 +921,53 @@ ecma_builtin_string_prototype_object_slice (ecma_string_t *get_string_val, /**<
   return ecma_make_string_value (new_str_p);
 } /* ecma_builtin_string_prototype_object_slice */
 
+/**
+ * The String.prototype object's 'at' routine
+ *
+ * See also:
+ *          ECMA-262 Stage 3 Draft
+ *
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+static ecma_value_t
+ecma_builtin_string_prototype_object_at (const ecma_value_t index, /**< index argument */
+                                         ecma_string_t *string_val) /**< this argument */
+{
+
+  const lit_utf8_size_t len = ecma_string_get_length (string_val);
+
+  ecma_number_t relative_index, k;
+
+  ecma_value_t index_to_integer = ecma_op_to_integer (index, &relative_index);
```

Comment:
For the "index_to_integer": please see the related comments above in the array's at implementation.